shared/components/MainHeader.js

```diff
@@ -201,13 +210,13 @@ export default class MainHeader extends Component {
                   this.props.categorizeItems('SHOW_ALL');

```

Comment:
why are you write show_all capitals? if you wan't to use small letter, plz standardization!